### PR TITLE
feat(performance): Update performance for Pools

### PR DESCRIPTION
**Pull request by onlychild**
## components/SearchModal
- audio.mp3 建议在播放的时候加载而不是preload

### defer google的语言包
``` javascript
    <link href="https://fonts.googleapis.com/css2?family=Kanit:wght@400;600&display=swap" rel="preload"  onload="this.onload=null; this.rel='stylesheet';"/>
```
### 删除无用的preconnect
```
<!-- <link rel="preconnect" href="%REACT_APP_NODE_PRODUCTION%" /> -->
<!-- <link rel="preconnect" href="https://fonts.gstatic.com" /> -->
```
### defer gtm
移动到script标签

### 移除ListsUpdater的预先加载
```
{/* <ListsUpdater /> */}
```

### 上下调整APP和updater标签的位置
```

```
### 现在的chunk包太大了，打包的时候建议拆包，defer 部分组件
收益：减少client的parse JS时间

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -54,5 +42,18 @@
...
     <div id="portal-root"></div>
+    <!-- GTAG -->
+    <script defer>
+      ;(function (w, d, s, l, i) {
+        w[l] = w[l] || []
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' })
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : ''
+        j.defer = true
```
Comment (chef-jojo) [inline public/index.html:53]: `defer` won't work for inline script, beside we already have defer in line 53.

File: src/components/SearchModal/CurrencySearch.tsx [hunk 2]
```diff
@@ -76,7 +74,7 @@ function CurrencySearch({
     (currency: Currency) => {
       onCurrencySelect(currency)
       if (audioPlay) {
-        swapSound.play()
+        new Audio('swap.mp3').play()
```
Comment (chef-jojo) [inline src/components/SearchModal/CurrencySearch.tsx:77]: Is a good idea to lazy load the audio file, but creating new instance on each swap action may cause some overhead 🤔

Comment (onlychild) [reply src/components/SearchModal/CurrencySearch.tsx:77]: just a way to lazy the audio files。
you can destroy the instance after play，in another way，you should to analysis how many times does caker to swap on each visit，overhead should base on the data as you know。for me，the load of the audio is useless due to my habit。

File: public/index.html (hunk 1 above)
Comment (onlychild) [reply public/index.html:53]: my fault...

Comment (onlychild) [reply public/index.html:53]: my fault...

**Comment by chef-jojo** [comment]
@onlychild swap sound lazy load merged in https://github.com/pancakeswap/pancake-frontend/pull/2955/files#diff-1097953b47b2ff58b6ef7d17c5d7f31b07df1ea13cc707980856eec63021ad12
I think this can be closed.
About the js chunk size issues, we are aware of that, but it requires a lot of rework and continuous progress to achieve it. Feel free to open issue to discuss 🙂

**Pull request closed without merging**